amgut/lib/data_access/ag_data_access.py

```diff
@@ -231,9 +231,8 @@ def registerHandoutKit(self, ag_login_id, supplied_kit_id):
             Non-UUID4 value sent as ag_login_id
         """
         with TRN:
-            uuid_check = UUID(ag_login_id)
-            if not uuid_check.version == 4:
-                raise ValueError("Non-UUID4 value passed: %s" % ag_login_id)
+            # make sure properly formatted UUID passed in
+            UUID(ag_login_id, version=4)
```

Comment:
Can you fix the raises section? It no longer raises a ValueError, it raises a UUID exception, right?